api/envoy/config/filter/http/admission_control/v2alpha/admission_control.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.admission_control.v2alpha;
+
+import "envoy/api/v2/core/base.proto";
+
+import "google/api/annotations.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.http.admission_control.v2alpha";
+option java_outer_classname = "AdmissionControlProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.filters.http.admission_control.v3alpha";
+
+// [#protodoc-title: Admission Control]
+// Admission Control :ref:`configuration overview
+// <config_http_filters_admission_control>`.
+// [#extension: envoy.filters.http.admission_control]
+
+message AdmissionControl {
```

Comment:
In terms of how to define/configure what success means, here is my suggestion:

I would define a oneof which contains a required message which defines how a request is considered successful. I think this will give us future flexibility to potentially add pluggable success criteria. In the near term I would have a message called something like DefaultSuccessCriteria which can be used to configure things like 5xx, gRPC, etc. Maybe let's try that type of shape and have a look at whether we feel it will be extensible enough or not?